examples/framework-alpine/astro.config.mjs

```diff
@@ -3,5 +3,5 @@ import { defineConfig } from 'astro/config';
 // https://astro.build/config
 export default defineConfig({
 	// No renderers are needed for AlpineJS support, just use Astro components!
-	renderers: [],
+	integrations: [],
```

Comment:
Nit: remove

Reply:
This is the only one I kept, I think its worth calling out that Alpine doesn't require an integration to run?